pytest_plt/tests/test_pytest.py

```diff
@@ -108,22 +108,20 @@ def test_plt_plots(testdir):
         assert os.path.exists(plot)
 
 
-@pytest.mark.parametrize('drop_len', [1, 3])
-def test_filename_drop_prefix(testdir, drop_len):
-    parts = ["package", "folder", "tests"]
-    drop_parts = parts[:drop_len]
+@pytest.mark.parametrize(
+    'prefix', ["package/", "package/folder/tests/"],
+)
+def test_filename_drop_prefix(testdir, prefix):
+    """Tests removing strings from the start of a plot filename.
 
-    for i in range(1, len(parts)+1):
-        testdir.mkpydir('/'.join(parts[:i]))
-
-    file_path = testdir.copy_example("test_plt.py")
-    file_path.rename('/'.join(parts + ["test_plt.py"]))
-
-    ini_source = r"""
-    [pytest]
-    plt_filename_drop =
-        """ + "".join(p + r"\." for p in drop_parts)
-    testdir.makeini(ini_source)
+    This is the most common use case of filename_drop.
+    """
+    copy_all_tests(testdir, prefix)
+    testdir.makeini("\n".join([
+        "[pytest]",
+        "plt_filename_drop =",
+        "    %s" % prefix.replace("/", r"\."),
+    ]))
```

Comment:
The only thing I don't like is that both times we're dropping the whole prefix here. I had intentionally set up the test to have a deep folder structure in both cases, but just drop part of it in the one case. It's not a big deal, though, since I think either way this case is well tested.

Reply:
Ah I see, that's wasn't clear when I read the test. Easy to change back.

Reply:
Done in 4d820f9f6b76e8f56a8ea9945dd6fff359a4dfec